src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/SingleFileApiTests.cs

```diff
@@ -67,6 +68,23 @@ public void AppContext_Deps_Files_Bundled_Self_Contained()
                 .NotHaveStdOutContaining("Microsoft.NETCore.App.deps.json");
         }
 
+        [Fact]
+        public void AppContext_Native_Search_Dirs_Contains_Bundle_And_Extraction_Dirs()
+        {
+            var fixture = sharedTestState.TestFixture.Copy();
+            Bundler bundler = BundleHelper.BundleApp(fixture, out string singleFile, BundleOptions.BundleNativeBinaries);
```

Comment:
Please add another test without any extraction - to validate that the logic works correctly

Reply:
Also once we merge this with the other PR the tests will need to change